Android/ThirdPartyAuth/app/src/main/java/com/bentley/sample/thirdpartyauth/LoginActivity.kt

```diff
@@ -0,0 +1,82 @@
+package com.bentley.sample.thirdpartyauth
+
+import android.content.res.Configuration
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Button
+import androidx.compose.runtime.livedata.observeAsState
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.bentley.sample.thirdpartyauth.ui.theme.ThirdPartyAuthTheme
+
+class LoginActivity : ComponentActivity() {
+
+    lateinit var loginViewModel: LoginViewModel
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        loginViewModel = (application as ThirdPartyAuthApplication).loginViewModel
+
+        setContent {
```

Comment:
Since I have no knowledge of Jetpack Compose UI, I have no idea if the UI code is reasonable. I'm going to assume it is. (@simihartstein resolve this comment after reading.)

Reply:
My understanding is that this is correct. This is my first foray into Jetpack Compose UI, so I could be wrong.